microraiden/microraiden/webui/index.html

```diff
@@ -152,7 +499,7 @@ <h5 class="col-sm-12 on-state on-state-opened">
           <div class="log">
             <h5 class="text-warning">No channel was found registered for this account on this website.</h5>
           </div>
-          <div class="row">
+        <!--  <div class="row">
```

Comment:
Sure you can comment out this part of the page? `div.transfer` is required for signature